### PR TITLE
ICU-21638 re-enable some of the ICU4J performance tests

### PR DESCRIPTION
**Pull request by sven-oly**
##### Checklist

- [x] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-21638
- [x] Required: The PR title must be prefixed with a JIRA Issue number. 
- [x] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [x] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [x] Issue accepted (done by Technical Committee after discussion)
- [x] Tests included, if applicable
- [x] API docs and/or User Guide docs changed or added, if applicable

### CODE REVIEW

File: icu4j/perf-tests/dateformatperf.pl [hunk 1]
```diff
@@ -16,6 +16,9 @@
 # Test class
 my $TESTCLASS = 'com.ibm.icu.dev.test.perf.DateFormatPerformanceTest';
 
+my $CLASSES = './out/bin:../tools/misc/out/bin/:../icu4j.jar';
+
+
```
Comment (gnrunge) [inline icu4j/perf-tests/dateformatperf.pl:21]: nit: one empty line enough? Any rule or precedence that requires two lines?
Here and all other files.

Comment (sven-oly) [reply icu4j/perf-tests/dateformatperf.pl]: Removed extra line

File: icu4j/perf-tests/README.txt [hunk 2]
```diff
@@ -4,7 +4,82 @@ Copyright (C) 2012, International Business Machines Corporation and others.  All
...
-      a) Statistics/Distribution.pm
-      b) Statistics/Descriptive.pm
-	c)  XML/LibXML.pm
+This directory includes a number of performance tests. Most are
+comparing ICU operations with built in Java functions. Many tests run
+numerous iterations with a variety of locales.
+
+Several tests create .html output files that can be opened and viewed in a web  browser.
+
+The collation test produces output in the terminal window Some are
```
Comment (gnrunge) [inline icu4j/perf-tests/README.txt:13]: ... terminal window. (full stop)

File: icu4j/perf-tests/README.txt [hunk 3]
```diff
@@ -4,7 +4,82 @@ Copyright (C) 2012, International Business Machines Corporation and others.  All
...
+individually via command line and others run as
```
Comment (gnrunge) [inline icu4j/perf-tests/README.txt:14]: Verb missing? E.g.: Some are <executed> individually

File: icu4j/perf-tests/decimalformatperf.pl [hunk 4]
```diff
@@ -16,6 +16,9 @@
 # Test class
 my $TESTCLASS = 'com.ibm.icu.dev.test.perf.DecimalFormatPerformanceTest';
 
+my $CLASSES = './out/bin:../tools/misc/out/bin/:../icu4j.jar';
+
+
```
Comment (gnrunge) [inline icu4j/perf-tests/decimalformatperf.pl:21]: One empty line too many.

File: icu4j/perf-tests/unicodesetperf.pl [hunk 5]
```diff
@@ -16,6 +16,9 @@
 # Test class
 my $TESTCLASS = 'com.ibm.icu.dev.test.perf.UnicodeSetPerf';
 
+my $CLASSES = './out/bin:../tools/misc/out/bin/:../icu4j.jar';
+
+
```
Comment (gnrunge) [inline icu4j/perf-tests/unicodesetperf.pl:21]: One empty line too many.

**Review by gnrunge — COMMENTED** [review]
Also squash the commits to one, please.

**Review by gnrunge — APPROVED** [review]

**Pull request merged**